game/js/game.js

```diff
@@ -29,6 +29,39 @@ $(document).ready(function () {
   }
 
 
+    $(document).keydown(function(e) {
+        switch(e.which) {
+            case 37: // left
+                $('#ludek').animate({
+                    'marginLeft': '-=30px'
```

Comment:
korytarzy miało być 9 po 55 pikseli. Przesunięcie zatem też będzie o 55 px. Rozumiem, że to wersja 1.0, więc najpierw chciałeś zawrzeć logikę, a zmienić to na 55px nie jest problemem.